server/storage/schema/confstate.go

```diff
@@ -49,6 +49,10 @@ func UnsafeConfStateFromBackend(lg *zap.Logger, tx backend.ReadTx) *raftpb.ConfS
 			zap.Int("number-of-key", len(keys)),
 		)
 	}
+	if len(vals[0]) == 0 {
```

Comment:
That's not a correct change, we cannot depend on changing server logic for implementing downgrades.